server/docs/resources.yml

```diff
@@ -7,6 +7,14 @@
       - authCookie: []
       - sudoToken: []
     parameters:
+      - in: query
+        name: account
+        description: |
+          Filter resources by authenticated user status.
+
+          **Note**: Published resources, belonging to the authenticated user, are available to him/her as drafts are only accessible to admin users.
```

Comment:
💡 I found this quite confusing, I think what you're trying to say is something like:
```suggestion
          **Note**: Published resources belonging to the authenticated user are available to all users; drafts are only accessible to admin users.
```